src/components/SearchBar/SearchBar.tsx

```diff
@@ -26,36 +26,46 @@ export const SearchBar = memo((props: ISearchBarProps) => {
     setInputValue(value);
   }, []);
 
-  const changeQuery = () => {
+  const changeQuery = useCallback(() => {
     setSearchParams((searchParams) => {
       searchParams.set(MainPageRoutes.PAGE, '1');
       return searchParams;
```

Comment:
I'm not sure that you need to return value from wrapped function, and I'm not sure that useCallback works properly